js/game.js

```diff
@@ -0,0 +1,77 @@
+/** @type {HTMLCanvasElement} */
+
+class Game {
+  constructor(ctx, width, height, player) {
+    this.ctx = ctx;
+    this.width = width;
+    this.height = height;
+    this.car = car;
+    this.intervalId = null;
+    this.frames = 0;
+    this.roadblock = [];
```

Comment:
Here, define the points that you are going to call in the `checkGameOver()`:
`this.points = 0;`